client/WEB-INF/classes/resources/messages_es.properties

```diff
@@ -808,7 +808,7 @@ label.skip.guide=He utilizado CloudStack anteriormente, saltar esta gu\u00eda
 label.snapshot=Instant\u00c3\u00a1nea
 label.snapshot.limits=instant\u00c3\u00a1neas L\u00c3\u00admites
 label.snapshot.name=Nombre de instant\u00c3\u00a1neas
-label.snapshot.schedule=Lista de instant\u00c3\u00a1neas
+label.snapshot.s=instant\u00c3\u00a1neas
```

Comment:
Yes, the value of label.snapshot.s key has been changes by mike (so need a new translation on each language):
https://git1-us-west.apache.org/repos/asf?p=cloudstack.git;a=blobdiff;f=client/WEB-INF/classes/resources/messages.properties;h=dcbf6c83a24a4cfd6977659d4f531c64c1786edb;hp=701582cb1321281bdede5daa0b5dc80e69375a35;hb=c0230273;hpb=406c2c00b267483ef6fe6c84b9a187458662cb9f

The git diff shown the label.snapshot.s key before label.snapshot.schedule because the messages_es.properties is in alphabetic order.